cmd/akash/cmd/root.go

```diff
@@ -217,6 +219,8 @@ func txCmd() *cobra.Command {
 	// add modules' tx commands
 	app.ModuleBasics().AddTxCommands(cmd)
 	cmd.PersistentFlags().String(flags.FlagChainID, "", "The network chain ID")
+	cmd.PersistentFlags().String(flags.FlagKeyringBackend, flags.DefaultKeyringBackend,
+		"Select keyring's backend (os|file|kwallet|pass|test)")
```

Comment:
This isn't added by sdk?

Reply:
Not currently, or some cobra nonsense is removing it from `tx`? Or do you mean in upstream changes?
```
akash tx -h
Transactions subcommands

Usage:
  akash tx [command]

Available Commands:
                      
                      
                      
  auth                Auth transaction subcommands
  bank                Bank transaction subcommands
  broadcast           Broadcast transactions generated offline
  crisis              Crisis transactions subcommands
  decode              Decode an binary encoded transaction string.
  deployment          Deployment transaction subcommands
  distribution        Distribution transactions subcommands
  encode              Encode transactions generated offline
  evidence            Evidence transaction subcommands
  gov                 Governance transactions subcommands
  market              Transaction subcommands
  multisign           Generate multisig signatures for transactions generated offline
  provider            Deployment transaction subcommands
  send                Send funds from one account to another. Note, the'--from' flag is
ignored as it is implied from [from_key_or_address].
  sign                Sign transactions generated offline
  sign-batch          Sign transaction batch files
  slashing            Slashing transaction subcommands
  staking             Staking transaction subcommands
  validate-signatures Validate transactions signatures
  vesting             Vesting transaction subcommands

Flags:
      --chain-id string   The network chain ID
  -h, --help              help for tx

Global Flags:
      --home string   directory for config and data (default "/home/ropes/.akash")
      --trace         print out full stack trace on errors

Additional help topics:
  akash tx upgrade       Upgrade transaction subcommands

Use "akash tx [command] --help" for more information about a command.
```

Reply:
not all tx commands take `--keyring-backend` - `broadcast` for instance.

these are added in individual commands like [this](https://github.com/ovrclk/akash/blob/26eb69c72b1a0a2ec2df8fc06c31b6dc361ca01a/x/deployment/client/cli/tx.go#L120)

Reply:
Alright, I'll circle back and run through the subcommands, add it only to the ones which hopefully need it.